shuttleofx_render/config.py

```diff
@@ -31,6 +31,8 @@
 
 globalOfxPluginPath = config.get("OFX_PATH", "globalOfxPluginPath")
 
+cacheMaxSize = config.getint('CACHE', 'maxSize')
+cleanCacheInterval = config.getint('CACHE', 'interval')
```

Comment:
Use `getfloat` allows to provide an interval < to 1h.
For testing, you can set 0.001 to invalidate the cache after 3 sec.